sigs.yaml

```diff
@@ -260,23 +260,23 @@ sigs:
     time: "19:00"
     tz: UTC
     frequency: biweekly
-    url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
+    url: no-zoomlink-match
```

Comment:
On these, I'd just leave it as the placeholder or reach out to the sig about updating it with the right time 👍

Reply:
Let me work on that

Reply:
@mrbobbytables reached out to the sig, got the links and I have now updated the PR.